### PR TITLE
api table added

### PR DESCRIPTION
**Pull request by rkchalasani**
api table added

### CODE REVIEW

**Comment by Maharshiharsha72** [comment]
@rkchalasani  why the branch names is apit ( what is the meaning of this )

File: apit/Apitable.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                <th scope="col">Title</th>
+                <th scope="col">Description</th>
+                <th scope="col">Price</th>
+                <th scope="col">Image</th>
+              </tr>
+            </thead>
+            <tbody id="table_body">
+              
+              <!-- <tr>
+                <th scope="row">2</th>
```
Comment (Maharshiharsha72) [inline apit/Apitable.html:46]: remove the commented code from these

File: apit/Apitable.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+fetch("https://fakestoreapi.com/products")
+  .then((data) => {
+    return data.json();
+  })
+  .then((objectData) => {
+    // console.log(objectData[0].title);
```
Comment (Maharshiharsha72) [inline apit/Apitable.js:6]: remove commented code

File: apit/Apitable.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+fetch("https://fakestoreapi.com/products")
+  .then((data) => {
+    return data.json();
```
Comment (Maharshiharsha72) [inline apit/Apitable.js:3]: can we not use only "then" once , instead of 2 times ???

File: apit/Apitable.html (hunk 1 above)
Comment (rkchalasani) [reply apit/Apitable.html:46]: sure maharshi i'll commit those changes

File: apit/Apitable.js (hunk 2 above)
Comment (rkchalasani) [reply apit/Apitable.js:6]: sure maharshi

**Pull request merged**